### DISCUSSION TITLE
8th Community Meeting 08/12/2022 - Last Meeting of this Year

### DISCUSSION THREAD

**Discussion by VojtechBartoska** [espressif/arduino-esp32#7544] [General]
Hello to all Arduino ESP32 Core Users 🚀 

**Meeting purpose,**
we appreciate all of your contribution and help and we would like to improve your User experience on this Project. 🤩

As a part of this improvement we would like to uncover our internal Development to you, have clear and transparent planning and also listen to your requests, ideas and valuable Feedback.

To fulfill this, we would like to held every Month Community Meeting where we can share main updates and also have an open discussion with you.

---

**💥Eight Community Meeting will take place on Thursday 08/12/2022 at 4 pm GMT+2 time zone. (1 hour length) We will use Google Meet.**

---

We want to have some idea about how many of you would be interested in joining these Meetings so instead of sharing directly link to Online Meeting itself, it will work by invitation.

**If you want to be added to upcoming Meeting and receive invitation please send an email to [vojtech.bartoska@espressif.com](mailto:vojtech.bartoska@espressif.com).** 

If you already participated on any Community Meeting, you are automatically invited to upcoming Meeting and it's up to you if you join us again or not.

---

### Agenda will be specified below in the comments, feel free to add any topic or idea :) .
1) Development updates
2) Year Summary
3) Overview of some major plans for next Year
4) Next episode of Community Meetings
.
.
.
any other proposed topic
---

See you on the Meeting!

**Comment by VojtechBartoska** [comment]
**Link to the Meeting is available below:**
8th Community Meeting Arduino ESP32 Core - Last Meeting this Year
Thursday, 8 December · 4:00 – 5:00pm
Google Meet joining info
Video call link: https://meet.google.com/jba-ahkg-tgz

**Comment by pedrominatel** [comment]
* Examples review and documentation
